.github/workflows/lint.yml

```diff
@@ -13,14 +13,14 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dprint/check@v2.2
         with:
-          dprint-version: '0.40.2' # selfup { "regex": "\\d[^']+", "script": "dprint --version | cut -d ' ' -f 2" }
+          dprint-version: '0.41.0' # selfup { "regex": "\\d[^']+", "script": "dprint --version | cut -d ' ' -f 2" }
```

Comment:
https://github.com/NixOS/nixpkgs/pull/260107
https://github.com/kachick/dotfiles/actions/runs/6514770322/job/17696404124?pr=337#step:6:877
https://github.com/NixOS/nixpkgs/blob/9b295ab713858412589e61c1f7c017e29699da0f/pkgs/development/tools/dprint/default.nix

failed in macos

Reply:
https://github.com/NixOS/nixpkgs/issues/101229

Reply:
https://github.com/NixOS/nixpkgs/pull/256590